Scan failure on invalid paths in dproj files

### Prerequisites

- [X] This bug is in SonarDelphi, not SonarQube or my Delphi code.
- [X] This bug has not already been reported.

### SonarDelphi version

1.5.0

### SonarQube version

10.5.1

### Issue description

EXECUTION FAILURE when scanning (1st time  am using SonarCube and SonarDelphi). Here the output

C:\Dev\w2w-main-dll>sonar-scanner.bat -X -D"sonar.projectKey=Jantest" -D"sonar.sources=." -D"sonar.host.url=http://localhost:9000" -D"sonar.token=sqp_8607858a1d68745fca4e8c2ec1cad1c4d62f6fc9"
12:11:29.946 INFO: Scanner configuration file: C:\Temp\sonarqube-10.5.1.90531\bin\..\conf\sonar-scanner.properties
12:11:29.953 INFO: Project root configuration file: NONE
12:11:29.966 INFO: SonarScanner 5.0.1.3006
12:11:29.966 INFO: Java 17.0.7 Eclipse Adoptium (64-bit)
12:11:29.966 INFO: Windows 10 10.0 amd64
12:11:30.035 DEBUG: keyStore is :
12:11:30.036 DEBUG: keyStore type is : pkcs12
12:11:30.037 DEBUG: keyStore provider is :
12:11:30.037 DEBUG: init keystore
12:11:30.037 DEBUG: init keymanager of type SunX509
12:11:30.135 DEBUG: Create: C:\Users\jdekkers\.sonar\cache
12:11:30.136 INFO: User cache: C:\Users\jdekkers\.sonar\cache
12:11:30.137 DEBUG: Create: C:\Users\jdekkers\.sonar\cache\_tmp
12:11:30.138 DEBUG: Extract sonar-scanner-api-batch in temp...
12:11:30.141 DEBUG: Get bootstrap index...
12:11:30.141 DEBUG: Download: http://localhost:9000/batch/index
12:11:30.172 DEBUG: Get bootstrap completed
12:11:30.173 DEBUG: Create isolated classloader...
12:11:30.185 DEBUG: Start temp cleaning...
12:11:30.191 DEBUG: Temp cleaning done
12:11:30.192 DEBUG: Execution getVersion
12:11:30.199 INFO: Analyzing on SonarQube server 10.5.1.90531
12:11:30.199 INFO: Default locale: "en_US", source code encoding: "windows-1252" (analysis is platform dependent)
12:11:30.200 DEBUG: Work directory: C:\Dev\w2w-main-dll\.scannerwork
12:11:30.200 DEBUG: Execution execute
12:11:30.374 DEBUG: Community 10.5.1.90531
12:11:30.532 INFO: Load global settings
12:11:30.560 DEBUG: GET 200 http://localhost:9000/api/settings/values.protobuf | time=27ms
12:11:30.577 INFO: Load global settings (done) | time=45ms
12:11:30.588 INFO: Server id: 147B411E-AY-DajVjxsStTV5upcR7
12:11:30.590 INFO: User cache: C:\Users\jdekkers\.sonar\cache
12:11:30.595 INFO: Loading required plugins
12:11:30.595 INFO: Load plugins index
12:11:30.602 DEBUG: GET 200 http://localhost:9000/api/plugins/installed | time=7ms
12:11:30.620 INFO: Load plugins index (done) | time=25ms
12:11:30.620 INFO: Load/download plugins
12:11:30.649 INFO: Load/download plugins (done) | time=29ms
12:11:30.649 DEBUG: Plugins not loaded because they are optional: [csharp, communitydelphi, flex, go, web, java, javascript, kotlin, php, ruby, sonarscala, vbnet]
12:11:30.663 DEBUG: Plugins loaded:
12:11:30.663 DEBUG:   * Python Code Quality and Security 4.17.0.14845 (python)
12:11:30.663 DEBUG:   * Clean as You Code 2.3.0.1782 (cayc)
12:11:30.664 DEBUG:   * XML Code Quality and Security 2.10.0.4108 (xml)
12:11:30.664 DEBUG:   * JaCoCo 1.3.0.1538 (jacoco)
12:11:30.664 DEBUG:   * IaC Code Quality and Security 1.27.0.9518 (iac)
12:11:30.664 DEBUG:   * Text Code Quality and Security 2.10.0.2188 (text)
12:11:30.752 DEBUG: register org.eclipse.jgit.util.FS$FileStoreAttributes$$Lambda$304/0x000000080127e3e8@15723761 with shutdown hook
12:11:30.830 INFO: Process project properties
12:11:30.838 INFO: Process project properties (done) | time=8ms
12:11:30.851 INFO: Project key: Jantest
12:11:30.851 INFO: Base dir: C:\Dev\w2w-main-dll
12:11:30.852 INFO: Working dir: C:\Dev\w2w-main-dll\.scannerwork
12:11:30.852 DEBUG: Project global encoding: windows-1252, default locale: en_US
12:11:30.860 INFO: Load project settings for component key: 'Jantest'
12:11:30.876 DEBUG: GET 200 http://localhost:9000/api/settings/values.protobuf?component=Jantest | time=16ms
12:11:30.878 INFO: Load project settings for component key: 'Jantest' (done) | time=18ms
12:11:30.893 DEBUG: Creating module hierarchy
12:11:30.893 DEBUG:   Init module 'Jantest'
12:11:30.894 DEBUG:     Base dir: C:\Dev\w2w-main-dll
12:11:30.894 DEBUG:     Working dir: C:\Dev\w2w-main-dll\.scannerwork
12:11:30.895 DEBUG:     Module global encoding: windows-1252, default locale: en_US
12:11:30.902 INFO: Load quality profiles
12:11:30.931 DEBUG: GET 200 http://localhost:9000/api/qualityprofiles/search.protobuf?project=Jantest | time=28ms
12:11:30.942 INFO: Load quality profiles (done) | time=40ms
12:11:30.967 INFO: Load active rules
12:11:31.007 DEBUG: GET 200 http://localhost:9000/api/rules/list.protobuf?qprofile=e15b95d3-df45-4dcd-81fd-aa2dfc59ca3a&ps=500&p=1 | time=40ms
12:11:31.096 DEBUG: GET 200 http://localhost:9000/api/rules/list.protobuf?qprofile=4de375a5-e81e-4c3e-bc78-47d8a229df0c&ps=500&p=1 | time=26ms
12:11:31.131 DEBUG: GET 200 http://localhost:9000/api/rules/list.protobuf?qprofile=05ce60e3-c42b-476e-913f-0aeaa8790b6e&ps=500&p=1 | time=30ms
12:11:31.139 DEBUG: GET 200 http://localhost:9000/api/rules/list.protobuf?qprofile=658fac45-905f-4c57-b013-73da5ae2dbf0&ps=500&p=1 | time=5ms
12:11:31.517 DEBUG: GET 200 http://localhost:9000/api/rules/list.protobuf?qprofile=1179d31b-f4f4-45ce-883e-51bfe8f057c7&ps=500&p=1 | time=378ms
12:11:31.873 DEBUG: GET 200 http://localhost:9000/api/rules/list.protobuf?qprofile=7cbe5ca6-6c6f-4a9b-b421-bd9ac77d8b85&ps=500&p=1 | time=330ms
12:11:31.937 DEBUG: GET 200 http://localhost:9000/api/rules/list.protobuf?qprofile=597d1f93-7925-4a11-9090-89bc0e966489&ps=500&p=1 | time=52ms
12:11:32.488 DEBUG: GET 200 http://localhost:9000/api/rules/list.protobuf?qprofile=0e64e82b-4df3-4558-9d68-f259238495e6&ps=500&p=1 | time=549ms
12:11:32.549 DEBUG: GET 200 http://localhost:9000/api/rules/list.protobuf?qprofile=0e64e82b-4df3-4558-9d68-f259238495e6&ps=500&p=2 | time=46ms
12:11:32.630 DEBUG: GET 200 http://localhost:9000/api/rules/list.protobuf?qprofile=64014a5c-9c71-406d-8938-bc381531aec0&ps=500&p=1 | time=78ms
12:11:32.671 DEBUG: GET 200 http://localhost:9000/api/rules/list.protobuf?qprofile=cdb9bc61-1842-4fb7-9063-8d9d575e56b7&ps=500&p=1 | time=39ms
12:11:32.707 DEBUG: GET 200 http://localhost:9000/api/rules/list.protobuf?qprofile=b699c5e8-7845-4361-8741-e5e0e438eb68&ps=500&p=1 | time=34ms
12:11:32.715 DEBUG: GET 200 http://localhost:9000/api/rules/list.protobuf?qprofile=00f91769-22b7-4b7b-b4e7-764ccec26767&ps=500&p=1 | time=7ms
12:11:32.728 DEBUG: GET 200 http://localhost:9000/api/rules/list.protobuf?qprofile=3782ee4d-819b-4082-8577-50a3843e9fcb&ps=500&p=1 | time=12ms
12:11:32.901 DEBUG: GET 200 http://localhost:9000/api/rules/list.protobuf?qprofile=f89eef42-287d-4789-986c-f49f789a7a82&ps=500&p=1 | time=172ms
12:11:32.947 DEBUG: GET 200 http://localhost:9000/api/rules/list.protobuf?qprofile=3c8980a2-7be8-4681-95a5-890f5fcb9fd3&ps=500&p=1 | time=43ms
12:11:32.953 DEBUG: GET 200 http://localhost:9000/api/rules/list.protobuf?qprofile=8cea3d35-a698-4170-b87b-ec655f46e1a2&ps=500&p=1 | time=4ms
12:11:33.078 DEBUG: GET 200 http://localhost:9000/api/rules/list.protobuf?qprofile=b286eb67-19d4-4686-8270-def4d665941c&ps=500&p=1 | time=124ms
12:11:33.109 DEBUG: GET 200 http://localhost:9000/api/rules/list.protobuf?qprofile=a29dee79-d1a0-4924-862b-334c1134cb42&ps=500&p=1 | time=28ms
12:11:33.260 DEBUG: GET 200 http://localhost:9000/api/rules/list.protobuf?qprofile=be110cbe-2774-4b15-aee9-8ae352da3aee&ps=500&p=1 | time=150ms
12:11:33.369 DEBUG: GET 200 http://localhost:9000/api/rules/list.protobuf?qprofile=2c520444-160e-4a2c-b1a9-64d1618b1455&ps=500&p=1 | time=105ms
12:11:33.407 DEBUG: GET 200 http://localhost:9000/api/rules/list.protobuf?qprofile=6928f487-ddca-4ee7-856b-a7534eddde1d&ps=500&p=1 | time=36ms
12:11:33.748 DEBUG: GET 200 http://localhost:9000/api/rules/list.protobuf?qprofile=d79275fe-3624-42c1-8884-5ecc40d8376b&ps=500&p=1 | time=340ms
12:11:33.938 DEBUG: GET 200 http://localhost:9000/api/rules/list.protobuf?qprofile=2580a57f-d017-427c-a2de-4dd090ac76ec&ps=500&p=1 | time=186ms
12:11:34.024 DEBUG: GET 200 http://localhost:9000/api/rules/list.protobuf?qprofile=cff44337-e1be-42a9-91c4-c36a08aac101&ps=500&p=1 | time=83ms
12:11:34.061 DEBUG: GET 200 http://localhost:9000/api/rules/list.protobuf?qprofile=6f6b2183-c7cf-4589-9e2d-13d29dbad68e&ps=500&p=1 | time=35ms
12:11:34.444 DEBUG: GET 200 http://localhost:9000/api/rules/list.protobuf?qprofile=ff480828-8a52-4584-833f-3d0b7c49d868&ps=500&p=1 | time=382ms
12:11:34.455 INFO: Load active rules (done) | time=3487ms
12:11:34.458 INFO: Load analysis cache
12:11:34.467 DEBUG: GET 200 http://localhost:9000/api/analysis_cache/get?project=Jantest | time=9ms
12:11:34.478 INFO: Load analysis cache (489 bytes) | time=20ms
12:11:34.491 DEBUG: GET 200 http://localhost:9000/api/languages/list | time=4ms
12:11:34.506 DEBUG: Declared patterns of language Kubernetes were converted to sonar.lang.patterns.kubernetes :
12:11:34.508 DEBUG: Declared patterns of language CSS were converted to sonar.lang.patterns.css : **/*.css,**/*.less,**/*.scss,**/*.sass
12:11:34.508 DEBUG: Declared patterns of language Scala were converted to sonar.lang.patterns.scala : **/*.scala
12:11:34.509 DEBUG: Declared patterns of language JSP were converted to sonar.lang.patterns.jsp : **/*.jsp,**/*.jspf,**/*.jspx
12:11:34.509 DEBUG: Declared patterns of language JavaScript were converted to sonar.lang.patterns.js : **/*.js,**/*.jsx,**/*.cjs,**/*.mjs,**/*.vue
12:11:34.509 DEBUG: Declared patterns of language Python were converted to sonar.lang.patterns.py : **/*.py
12:11:34.510 DEBUG: Declared patterns of language Docker were converted to sonar.lang.patterns.docker : **/Dockerfile,**/*.dockerfile
12:11:34.511 DEBUG: Declared patterns of language Java were converted to sonar.lang.patterns.java : **/*.java,**/*.jav
12:11:34.512 DEBUG: Declared patterns of language HTML were converted to sonar.lang.patterns.web : **/*.html,**/*.xhtml,**/*.cshtml,**/*.vbhtml,**/*.aspx,**/*.ascx,**/*.rhtml,**/*.erb,**/*.shtm,**/*.shtml,**/*.cmp,**/*.twig
12:11:34.512 DEBUG: Declared patterns of language Flex were converted to sonar.lang.patterns.flex : **/*.as
12:11:34.513 DEBUG: Declared patterns of language XML were converted to sonar.lang.patterns.xml : **/*.xml,**/*.xsd,**/*.xsl,**/*.config
12:11:34.513 DEBUG: Declared patterns of language JSON were converted to sonar.lang.patterns.json : **/*.json
12:11:34.513 DEBUG: Declared patterns of language Text were converted to sonar.lang.patterns.text :
12:11:34.513 DEBUG: Declared patterns of language VB.NET were converted to sonar.lang.patterns.vbnet : **/*.vb
12:11:34.513 DEBUG: Declared patterns of language CloudFormation were converted to sonar.lang.patterns.cloudformation :
12:11:34.513 DEBUG: Declared patterns of language YAML were converted to sonar.lang.patterns.yaml : **/*.yaml,**/*.yml
12:11:34.514 DEBUG: Declared patterns of language Delphi were converted to sonar.lang.patterns.delphi : **/*.pas,**/*.dpr,**/*.dpk
12:11:34.514 DEBUG: Declared patterns of language Go were converted to sonar.lang.patterns.go : **/*.go
12:11:34.514 DEBUG: Declared patterns of language Kotlin were converted to sonar.lang.patterns.kotlin : **/*.kt,**/*.kts
12:11:34.514 DEBUG: Declared patterns of language Secrets were converted to sonar.lang.patterns.secrets :
12:11:34.514 DEBUG: Declared patterns of language Ruby were converted to sonar.lang.patterns.ruby : **/*.rb
12:11:34.514 DEBUG: Declared patterns of language C# were converted to sonar.lang.patterns.cs : **/*.cs,**/*.razor
12:11:34.515 DEBUG: Declared patterns of language PHP were converted to sonar.lang.patterns.php : **/*.php,**/*.php3,**/*.php4,**/*.php5,**/*.phtml,**/*.inc
12:11:34.515 DEBUG: Declared patterns of language Terraform were converted to sonar.lang.patterns.terraform : **/*.tf
12:11:34.515 DEBUG: Declared patterns of language AzureResourceManager were converted to sonar.lang.patterns.azureresourcemanager : **/*.bicep
12:11:34.515 DEBUG: Declared patterns of language TypeScript were converted to sonar.lang.patterns.ts : **/*.ts,**/*.tsx,**/*.cts,**/*.mts
12:11:34.523 INFO: Preprocessing files...
12:11:34.534 DEBUG: loading config FileBasedConfig[C:\Users\jdekkers\.config\jgit\config]
12:11:34.536 DEBUG: loading config FileBasedConfig[C:\Users\jdekkers\.config\git\config]
12:11:34.537 DEBUG: loading config UserConfigFile[C:\Users\jdekkers\.gitconfig]
12:11:34.610 DEBUG: 362 non excluded files in this Git repository

~~

~~

12:11:34.939 INFO: 2 languages detected in 360 preprocessed files
12:11:34.939 INFO: 693 files ignored because of scm ignore settings
12:11:34.942 INFO: Loading plugins for detected languages
12:11:34.942 DEBUG: Detected languages: [delphi, php]
12:11:34.944 INFO: Load/download plugins
12:11:34.949 INFO: Load/download plugins (done) | time=5ms
12:11:34.950 DEBUG: Optional language-specific plugins not loaded: [csharp, flex, go, java, javascript, kotlin, ruby, sonarscala, vbnet]
12:11:34.958 DEBUG: Plugins loaded:
12:11:34.958 DEBUG:   * PHP Code Quality and Security 3.35.0.11659 (php)
12:11:34.958 DEBUG:   * HTML Code Quality and Security 3.15.0.5107 (web)
12:11:34.958 DEBUG:   * Delphi 1.5.0 (communitydelphi)
12:11:35.037 INFO: Inconsistent constructor declaration on bean with name 'org.sonarsource.scanner.api.internal.IsolatedClassloader@6fb0d3ed-org.sonar.scanner.issue.IssueFilters': single autowire-marked constructor flagged as optional - this constructor is effectively required since there is no default constructor to fall back to: public org.sonar.scanner.issue.IssueFilters(org.sonar.api.batch.fs.internal.DefaultInputProject)
12:11:35.053 INFO: Load project repositories
12:11:35.070 DEBUG: GET 200 http://localhost:9000/batch/project.protobuf?key=Jantest | time=16ms
12:11:35.080 INFO: Load project repositories (done) | time=27ms
12:11:35.094 DEBUG: Available languages:
12:11:35.096 DEBUG:   * PHP => "php"
12:11:35.096 DEBUG:   * HTML => "web"
12:11:35.096 DEBUG:   * JSP => "jsp"
12:11:35.096 DEBUG:   * Delphi => "delphi"
12:11:35.096 DEBUG:   * Python => "py"
12:11:35.096 DEBUG:   * XML => "xml"
12:11:35.096 DEBUG:   * Terraform => "terraform"
12:11:35.096 DEBUG:   * CloudFormation => "cloudformation"
12:11:35.096 DEBUG:   * Kubernetes => "kubernetes"
12:11:35.096 DEBUG:   * Docker => "docker"
12:11:35.096 DEBUG:   * AzureResourceManager => "azureresourcemanager"
12:11:35.096 DEBUG:   * YAML => "yaml"
12:11:35.096 DEBUG:   * JSON => "json"
12:11:35.096 DEBUG:   * Text => "text"
12:11:35.096 DEBUG:   * Secrets => "secrets"
12:11:35.097 INFO: Indexing files...
12:11:35.098 INFO: Project configuration:
12:11:35.102 DEBUG: '.gitignore' indexed with no language
12:11:35.106 DEBUG: 'debugss.pas' indexed with language 'delphi'
12:11:35.107 DEBUG: 'DelphiXEUnits\Web.HTTPApp.pas' indexed with language 'delphi'
12:11:35.107 DEBUG: 'DelphiXEUnits\Web.HTTPAppPete.pas' indexed with language 'delphi'
12:11:35.107 DEBUG: 'DKIM Notes.txt' indexed with no language
12:11:35.130 DEBUG: 'IdCompilerDefines.inc' generated metadata with charset 'windows-1252'
12:11:35.140 DEBUG: 'IdCompilerDefines.inc' indexed with language 'php'
12:11:35.140 DEBUG: 'L' indexed with no language
12:11:35.141 DEBUG: 'MDWebXE7-x\dclWebSess.dcr' indexed with no language
12:11:35.141 DEBUG: 'MDWebXE7-x\dclWebSess.dof' indexed with no language
12:11:35.142 DEBUG: 'MDWebXE7-x\dclWebSess.dpk' indexed with language 'delphi'
12:11:35.144 DEBUG: 'MDWebXE7-x\dclWebSess.res' indexed with no language
12:11:35.145 DEBUG: 'MDWebXE7-x\MDBTree.pas' indexed with language 'delphi'
12:11:35.145 DEBUG: 'MDWebXE7-x\mdChart.bmp' indexed with no language
12:11:35.146 DEBUG: 'MDWebXE7-x\MDDataSetPageProducer.pas' indexed with language 'delphi'
12:11:35.146 DEBUG: 'MDWebXE7-x\MDDataSetTableProducer.pas' indexed with language 'delphi'
12:11:35.147 DEBUG: 'MDWebXE7-x\mdpage.bmp' indexed with no language
12:11:35.147 DEBUG: 'MDWebXE7-x\MDPageProducer.pas' indexed with language 'delphi'
12:11:35.147 DEBUG: 'MDWebXE7-x\mdQuery.bmp' indexed with no language
12:11:35.147 DEBUG: 'MDWebXE7-x\MDQueryTableProducer.pas' indexed with language 'delphi'
12:11:35.148 DEBUG: 'MDWebXE7-x\mdSess.bmp' indexed with no language
12:11:35.148 DEBUG: 'MDWebXE7-x\MDSessMgr.pas' indexed with language 'delphi'
12:11:35.149 DEBUG: 'MDWebXE7-x\mdTable.bmp' indexed with no language
12:11:35.149 DEBUG: 'MDWebXE7-x\MDWebChart.pas' indexed with language 'delphi'
12:11:35.149 DEBUG: 'MDWebXE7-x\WebSessXE7.dpk' indexed with language 'delphi'
12:11:35.150 DEBUG: 'MDWebXE7-x\WebSessXE7.dproj' indexed with no language
12:11:35.150 DEBUG: 'MDWebXE7-x\WebSessXE7.res' indexed with no language
12:11:35.150 DEBUG: 'ModelSupport_w2w\default.txaPackage' indexed with no language
12:11:35.150 DEBUG: 'ModelSupport_w2w\default.txvpck' indexed with no language
12:11:35.150 DEBUG: 'ModelSupport_w2w\w2w\default.txaPackage' indexed with no language
12:11:35.151 DEBUG: 'ModelSupport_w2w\w2w\default.txvpck' indexed with no language
12:11:35.151 DEBUG: 'ModelSupport_w2w\w2wBasic\default.txaPackage' indexed with no language
12:11:35.152 DEBUG: 'ModelSupport_w2w\w2wBasic\default.txvpck' indexed with no language
12:11:35.153 DEBUG: 'ModelSupport_w2w\w2wConst\default.txaPackage' indexed with no language
12:11:35.153 DEBUG: 'ModelSupport_w2w\w2wConst\default.txvpck' indexed with no language
12:11:35.154 DEBUG: 'ModelSupport_w2w\w2wCore\default.txaPackage' indexed with no language
12:11:35.154 DEBUG: 'ModelSupport_w2w\w2wCore\default.txvpck' indexed with no language
12:11:35.155 DEBUG: 'ModelSupport_w2w\w2wEmailUtils\default.txaPackage' indexed with no language
12:11:35.155 DEBUG: 'ModelSupport_w2w\w2wEmailUtils\default.txvpck' indexed with no language
12:11:35.156 DEBUG: 'ModelSupport_w2w\w2wEmpEntry\default.txaPackage' indexed with no language
12:11:35.157 DEBUG: 'ModelSupport_w2w\w2wEmpEntry\default.txvpck' indexed with no language
12:11:35.163 DEBUG: 'ModelSupport_w2w\w2wHash\default.txaPackage' indexed with no language
12:11:35.163 DEBUG: 'ModelSupport_w2w\w2wHash\default.txvpck' indexed with no language
12:11:35.164 DEBUG: 'ModelSupport_w2w\w2wMain\default.txaPackage' indexed with no language
12:11:35.164 DEBUG: 'ModelSupport_w2w\w2wMain\default.txvpck' indexed with no language
12:11:35.165 DEBUG: 'ModelSupport_w2w\w2wTypes\default.txaPackage' indexed with no language
12:11:35.165 DEBUG: 'ModelSupport_w2w\w2wTypes\default.txvpck' indexed with no language
12:11:35.166 DEBUG: 'NameValueList.pas' indexed with language 'delphi'
12:11:35.166 DEBUG: 'pubsave.txt' indexed with no language
12:11:35.166 DEBUG: 'PUnitTesting.dpr' indexed with language 'delphi'
12:11:35.167 DEBUG: 'PUnitTesting.dproj' indexed with no language
12:11:35.167 DEBUG: 'PUnitTesting.otares' indexed with no language
12:11:35.168 DEBUG: 'PUnitTesting.res' indexed with no language
12:11:35.168 DEBUG: 'PUnitTesting_Icon.ico' indexed with no language
12:11:35.169 DEBUG: 'SchedDB4.pas' indexed with language 'delphi'
12:11:35.169 DEBUG: 'SchedDB5.pas' indexed with language 'delphi'
12:11:35.169 DEBUG: 'SendMailThrd.pas' indexed with language 'delphi'
12:11:35.170 DEBUG: 'SFMExport499\DEMOS\link.txt' indexed with no language
12:11:35.171 DEBUG: 'SFMExport499\file_id.diz' indexed with no language
12:11:35.172 DEBUG: 'SFMExport499\license.txt' indexed with no language
12:11:35.172 DEBUG: 'SFMExport499\RAD2010\ExCnst.hpp' indexed with no language
12:11:35.173 DEBUG: 'SFMExport499\RAD2010\ExportDS.dcr' indexed with no language
12:11:35.177 DEBUG: 'SFMExport499\RAD2010\ExportDS.dfm' indexed with no language
12:11:35.178 DEBUG: 'SFMExport499\RAD2010\ExportDS.hpp' indexed with no language
12:11:35.179 DEBUG: 'SFMExport499/RAD2010/SME.INC' generated metadata with charset 'windows-1252'
12:11:35.181 DEBUG: 'SFMExport499\RAD2010\SME.INC' indexed with language 'php'
12:11:35.181 DEBUG: 'SFMExport499\RAD2010\SME.RES' indexed with no language
12:11:35.182 DEBUG: 'SFMExport499\RAD2010\SME2ADO.hpp' indexed with no language
12:11:35.182 DEBUG: 'SFMExport499\RAD2010\SME2BDE.hpp' indexed with no language
12:11:35.183 DEBUG: 'SFMExport499\RAD2010\SME2Cell.hpp' indexed with no language
12:11:35.183 DEBUG: 'SFMExport499\RAD2010\SME2CLP.hpp' indexed with no language
12:11:35.183 DEBUG: 'SFMExport499\RAD2010\SME2DBF.hpp' indexed with no language
12:11:35.184 DEBUG: 'SFMExport499\RAD2010\SME2DIF.hpp' indexed with no language
12:11:35.185 DEBUG: 'SFMExport499\RAD2010\SME2DS.hpp' indexed with no language
12:11:35.185 DEBUG: 'SFMExport499\RAD2010\SME2HTML.hpp' indexed with no language
12:11:35.185 DEBUG: 'SFMExport499\RAD2010\SME2LDIF.hpp' indexed with no language
12:11:35.186 DEBUG: 'SFMExport499\RAD2010\SME2OLE.hpp' indexed with no language
12:11:35.186 DEBUG: 'SFMExport499\RAD2010\SME2PDF.hpp' indexed with no language
12:11:35.187 DEBUG: 'SFMExport499\RAD2010\SME2Pump.hpp' indexed with no language
12:11:35.188 DEBUG: 'SFMExport499\RAD2010\SME2PumpADO.hpp' indexed with no language
12:11:35.188 DEBUG: 'SFMExport499\RAD2010\SME2PumpBDE.hpp' indexed with no language
12:11:35.189 DEBUG: 'SFMExport499\RAD2010\SME2PumpDOA.hpp' indexed with no language
12:11:35.193 DEBUG: 'SFMExport499\RAD2010\SME2PumpIBX.hpp' indexed with no language
12:11:35.193 DEBUG: 'SFMExport499\RAD2010\SME2PumpMDB.hpp' indexed with no language
12:11:35.193 DEBUG: 'SFMExport499\RAD2010\SME2RTF.hpp' indexed with no language
12:11:35.194 DEBUG: 'SFMExport499\RAD2010\SME2SPSS.hpp' indexed with no language
12:11:35.194 DEBUG: 'SFMExport499\RAD2010\SME2SQL.hpp' indexed with no language
12:11:35.195 DEBUG: 'SFMExport499\RAD2010\SME2SYLK.hpp' indexed with no language
12:11:35.195 DEBUG: 'SFMExport499\RAD2010\SME2TXT.hpp' indexed with no language
12:11:35.195 DEBUG: 'SFMExport499\RAD2010\SME2WKS.hpp' indexed with no language
12:11:35.196 DEBUG: 'SFMExport499\RAD2010\SME2WQ.hpp' indexed with no language
12:11:35.196 DEBUG: 'SFMExport499\RAD2010\SME2XLS.hpp' indexed with no language
12:11:35.196 DEBUG: 'SFMExport499\RAD2010\SME2XML.hpp' indexed with no language
12:11:35.196 DEBUG: 'SFMExport499\RAD2010\SMEEngCX.hpp' indexed with no language
12:11:35.197 DEBUG: 'SFMExport499\RAD2010\SMEEngDB.hpp' indexed with no language
12:11:35.197 DEBUG: 'SFMExport499\RAD2010\SMEEngDC.hpp' indexed with no language
12:11:35.197 DEBUG: 'SFMExport499\RAD2010\SMEEngDOA.hpp' indexed with no language
12:11:35.197 DEBUG: 'SFMExport499\RAD2010\SMEEngDX.hpp' indexed with no language
12:11:35.198 DEBUG: 'SFMExport499\RAD2010\SMEEngEH.hpp' indexed with no language
12:11:35.198 DEBUG: 'SFMExport499\RAD2010\SMEEngFxDC.hpp' indexed with no language
12:11:35.198 DEBUG: 'SFMExport499\RAD2010\SMEEngIBO.hpp' indexed with no language
12:11:35.199 DEBUG: 'SFMExport499\RAD2010\SMEEngine.hpp' indexed with no language
12:11:35.199 DEBUG: 'SFMExport499\RAD2010\SMEEngLV.hpp' indexed with no language
12:11:35.199 DEBUG: 'SFMExport499\RAD2010\SMEEngRZLV.hpp' indexed with no language
12:11:35.199 DEBUG: 'SFMExport499\RAD2010\SMEEngSG.hpp' indexed with no language
12:11:35.200 DEBUG: 'SFMExport499\RAD2010\SMEEngSM.hpp' indexed with no language
12:11:35.200 DEBUG: 'SFMExport499\RAD2010\SMEEngSMI.hpp' indexed with no language
12:11:35.200 DEBUG: 'SFMExport499\RAD2010\SMEEngStrings.hpp' indexed with no language
12:11:35.200 DEBUG: 'SFMExport499\RAD2010\SMEEngVG.hpp' indexed with no language
12:11:35.201 DEBUG: 'SFMExport499\RAD2010\SMEEngWW.hpp' indexed with no language
12:11:35.201 DEBUG: 'SFMExport499\RAD2010\SMEMIME.hpp' indexed with no language
12:11:35.201 DEBUG: 'SFMExport499\RAD2010\SMEMonitor.dfm' indexed with no language
12:11:35.201 DEBUG: 'SFMExport499\RAD2010\SMEMonitor.hpp' indexed with no language
12:11:35.202 DEBUG: 'SFMExport499\RAD2010\SMEReg.hpp' indexed with no language
12:11:35.202 DEBUG: 'SFMExport499\RAD2010\SMESave.dfm' indexed with no language
12:11:35.202 DEBUG: 'SFMExport499\RAD2010\SMESave.hpp' indexed with no language
12:11:35.202 DEBUG: 'SFMExport499\RAD2010\SMESpecs.dfm' indexed with no language
12:11:35.202 DEBUG: 'SFMExport499\RAD2010\SMESpecs.hpp' indexed with no language
12:11:35.203 DEBUG: 'SFMExport499\RAD2010\SMEStat.dfm' indexed with no language
12:11:35.203 DEBUG: 'SFMExport499\RAD2010\SMEStat.hpp' indexed with no language
12:11:35.203 DEBUG: 'SFMExport499\RAD2010\SMEUtils.hpp' indexed with no language
12:11:35.204 DEBUG: 'SFMExport499\RAD2010\SMEWiz.dfm' indexed with no language
12:11:35.204 DEBUG: 'SFMExport499\RAD2010\SMEWiz.hpp' indexed with no language
12:11:35.204 DEBUG: 'SFMExport499\RAD2010\smew_cat.res' indexed with no language
12:11:35.205 DEBUG: 'SFMExport499\RAD2010\SMExportD2010.hpp' indexed with no language
12:11:35.210 DEBUG: 'SFMExport499\RAD2010\SMExportD2010.res' indexed with no language
12:11:35.210 DEBUG: 'SFMExport499\readme.txt' indexed with no language
12:11:35.210 DEBUG: 'SFMExport499\sme.htm' indexed with no language
12:11:35.210 DEBUG: 'SFMExport499\sme.inf' indexed with no language
12:11:35.211 DEBUG: 'SFMExport499\smexport.cnt' indexed with no language
12:11:35.211 DEBUG: 'SFMExport499\smexport.hlp' indexed with no language
12:11:35.211 DEBUG: 'SFMExport499\sme_faq.htm' indexed with no language
12:11:35.211 DEBUG: 'SFMExport499\Sources\ExCnst.pas' indexed with language 'delphi'
12:11:35.211 DEBUG: 'SFMExport499\Sources\ExportDS.dcr' indexed with no language
12:11:35.211 DEBUG: 'SFMExport499\Sources\ExportDS.dfm' indexed with no language
12:11:35.212 DEBUG: 'SFMExport499\Sources\ExportDS.pas' indexed with language 'delphi'
12:11:35.212 DEBUG: 'SFMExport499\Sources\RESOURCES\BRA\ExCnst.pas' indexed with language 'delphi'
12:11:35.212 DEBUG: 'SFMExport499\Sources\RESOURCES\BRA\ExCnst1.pas' indexed with language 'delphi'
12:11:35.212 DEBUG: 'SFMExport499\Sources\RESOURCES\BRA\ExCnst3.pas' indexed with language 'delphi'
12:11:35.213 DEBUG: 'SFMExport499\Sources\RESOURCES\CHI_BIG51\ExCnst.pas' indexed with language 'delphi'
12:11:35.213 DEBUG: 'SFMExport499\Sources\RESOURCES\CHI_BIG52\ExCnst.pas' indexed with language 'delphi'
12:11:35.213 DEBUG: 'SFMExport499\Sources\RESOURCES\CHI_SIMP1\ExCnst.pas' indexed with language 'delphi'
12:11:35.213 DEBUG: 'SFMExport499\Sources\RESOURCES\CHI_SIMP2\ExCnst.pas' indexed with language 'delphi'
12:11:35.214 DEBUG: 'SFMExport499\Sources\RESOURCES\CZ\ExCnst.pas' indexed with language 'delphi'
12:11:35.214 DEBUG: 'SFMExport499\Sources\RESOURCES\DAN\ExCnst.pas' indexed with language 'delphi'
12:11:35.214 DEBUG: 'SFMExport499\Sources\RESOURCES\ENG\ExCnst.pas' indexed with language 'delphi'
12:11:35.215 DEBUG: 'SFMExport499\Sources\RESOURCES\FRA\ExCnst.pas' indexed with language 'delphi'
12:11:35.215 DEBUG: 'SFMExport499\Sources\RESOURCES\FRA\ExCnst2.pas' indexed with language 'delphi'
12:11:35.215 DEBUG: 'SFMExport499\Sources\RESOURCES\GER\ExCnst.pas' indexed with language 'delphi'
12:11:35.215 DEBUG: 'SFMExport499\Sources\RESOURCES\GRE\EXCNST.PAS' indexed with language 'delphi'
12:11:35.216 DEBUG: 'SFMExport499\Sources\RESOURCES\HEB\ExCnst.pas' indexed with language 'delphi'
12:11:35.216 DEBUG: 'SFMExport499\Sources\RESOURCES\ID\ExCnst.pas' indexed with language 'delphi'
12:11:35.216 DEBUG: 'SFMExport499\Sources\RESOURCES\ITA\ExCnst.pas' indexed with language 'delphi'
12:11:35.216 DEBUG: 'SFMExport499\Sources\RESOURCES\NLD\ExCnst.pas' indexed with language 'delphi'
12:11:35.217 DEBUG: 'SFMExport499\Sources\RESOURCES\NLD\ExCnst2.pas' indexed with language 'delphi'
12:11:35.217 DEBUG: 'SFMExport499\Sources\RESOURCES\NLD\ExCnstNI.pas' indexed with language 'delphi'
12:11:35.217 DEBUG: 'SFMExport499\Sources\RESOURCES\NOR\ExCnst.pas' indexed with language 'delphi'
12:11:35.217 DEBUG: 'SFMExport499\Sources\RESOURCES\POL\ExCnst.pas' indexed with language 'delphi'
12:11:35.218 DEBUG: 'SFMExport499\Sources\RESOURCES\POR\ExCnst.pas' indexed with language 'delphi'
12:11:35.218 DEBUG: 'SFMExport499\Sources\RESOURCES\POR\ExCnst1.pas' indexed with language 'delphi'
12:11:35.218 DEBUG: 'SFMExport499\Sources\RESOURCES\ROM\ExCnst.pas' indexed with language 'delphi'
12:11:35.218 DEBUG: 'SFMExport499\Sources\RESOURCES\RUS\ExCnst.pas' indexed with language 'delphi'
12:11:35.219 DEBUG: 'SFMExport499\Sources\RESOURCES\SLO1\ExCnst.pas' indexed with language 'delphi'
12:11:35.219 DEBUG: 'SFMExport499\Sources\RESOURCES\SLO2\ExCnst.pas' indexed with language 'delphi'
12:11:35.219 DEBUG: 'SFMExport499\Sources\RESOURCES\SPA\ExCnst.pas' indexed with language 'delphi'
12:11:35.220 DEBUG: 'SFMExport499\Sources\RESOURCES\SPA\ExCnstSPA.pas' indexed with language 'delphi'
12:11:35.226 DEBUG: 'SFMExport499\Sources\RESOURCES\SWE\ExCnst.pas' indexed with language 'delphi'
12:11:35.226 DEBUG: 'SFMExport499\Sources\RESOURCES\TUR\ExCnst.pas' indexed with language 'delphi'
12:11:35.227 DEBUG: 'SFMExport499\Sources\RESOURCES\TUR1\ExCnst.pas' indexed with language 'delphi'
12:11:35.227 DEBUG: 'SFMExport499\Sources\RESOURCES\TUR2\ExCnst.pas' indexed with language 'delphi'
12:11:35.227 DEBUG: 'SFMExport499\Sources\RESOURCES\UKR\ExCnst.pas' indexed with language 'delphi'
12:11:35.228 DEBUG: 'SFMExport499/Sources/SME.INC' generated metadata with charset 'windows-1252'
12:11:35.229 DEBUG: 'SFMExport499\Sources\SME.INC' indexed with language 'php'
12:11:35.229 DEBUG: 'SFMExport499\Sources\SME.RES' indexed with no language
12:11:35.229 DEBUG: 'SFMExport499\Sources\SME2ADO.pas' indexed with language 'delphi'
12:11:35.230 DEBUG: 'SFMExport499\Sources\SME2BDE.pas' indexed with language 'delphi'
12:11:35.230 DEBUG: 'SFMExport499\Sources\SME2Cell.pas' indexed with language 'delphi'
12:11:35.230 DEBUG: 'SFMExport499\Sources\SME2CLP.pas' indexed with language 'delphi'
12:11:35.230 DEBUG: 'SFMExport499\Sources\SME2DBF.pas' indexed with language 'delphi'
12:11:35.231 DEBUG: 'SFMExport499\Sources\SME2DIF.pas' indexed with language 'delphi'
12:11:35.231 DEBUG: 'SFMExport499\Sources\SME2DS.pas' indexed with language 'delphi'
12:11:35.231 DEBUG: 'SFMExport499\Sources\SME2HTML.pas' indexed with language 'delphi'
12:11:35.231 DEBUG: 'SFMExport499\Sources\SME2LDIF.pas' indexed with language 'delphi'
12:11:35.232 DEBUG: 'SFMExport499\Sources\SME2OLE.pas' indexed with language 'delphi'
12:11:35.232 DEBUG: 'SFMExport499\Sources\SME2PDF.pas' indexed with language 'delphi'
12:11:35.232 DEBUG: 'SFMExport499\Sources\SME2Pump.pas' indexed with language 'delphi'
12:11:35.232 DEBUG: 'SFMExport499\Sources\SME2PumpADO.pas' indexed with language 'delphi'
12:11:35.232 DEBUG: 'SFMExport499\Sources\SME2PumpBDE.pas' indexed with language 'delphi'
12:11:35.233 DEBUG: 'SFMExport499\Sources\SME2PumpDOA.pas' indexed with language 'delphi'
12:11:35.233 DEBUG: 'SFMExport499\Sources\SME2PumpIBX.pas' indexed with language 'delphi'
12:11:35.233 DEBUG: 'SFMExport499\Sources\SME2PumpMDB.pas' indexed with language 'delphi'
12:11:35.233 DEBUG: 'SFMExport499\Sources\SME2RTF.pas' indexed with language 'delphi'
12:11:35.233 DEBUG: 'SFMExport499\Sources\SME2SPSS.pas' indexed with language 'delphi'
12:11:35.233 DEBUG: 'SFMExport499\Sources\SME2SQL.pas' indexed with language 'delphi'
12:11:35.233 DEBUG: 'SFMExport499\Sources\SME2SYLK.pas' indexed with language 'delphi'
12:11:35.233 DEBUG: 'SFMExport499\Sources\SME2TXT.pas' indexed with language 'delphi'
12:11:35.233 DEBUG: 'SFMExport499\Sources\SME2WKS.pas' indexed with language 'delphi'
12:11:35.234 DEBUG: 'SFMExport499\Sources\SME2WQ.pas' indexed with language 'delphi'
12:11:35.234 DEBUG: 'SFMExport499\Sources\SME2XLS.pas' indexed with language 'delphi'
12:11:35.234 DEBUG: 'SFMExport499\Sources\SME2XML.pas' indexed with language 'delphi'
12:11:35.235 DEBUG: 'SFMExport499\Sources\SMEEngCX.pas' indexed with language 'delphi'
12:11:35.235 DEBUG: 'SFMExport499\Sources\SMEEngDB.pas' indexed with language 'delphi'
12:11:35.235 DEBUG: 'SFMExport499\Sources\SMEEngDC.pas' indexed with language 'delphi'
12:11:35.235 DEBUG: 'SFMExport499\Sources\SMEEngDOA.pas' indexed with language 'delphi'
12:11:35.236 DEBUG: 'SFMExport499\Sources\SMEEngDX.pas' indexed with language 'delphi'
12:11:35.242 DEBUG: 'SFMExport499\Sources\SMEEngEH.pas' indexed with language 'delphi'
12:11:35.242 DEBUG: 'SFMExport499\Sources\SMEEngFxDC.pas' indexed with language 'delphi'
12:11:35.243 DEBUG: 'SFMExport499\Sources\SMEEngIBO.pas' indexed with language 'delphi'
12:11:35.243 DEBUG: 'SFMExport499\Sources\SMEEngine.pas' indexed with language 'delphi'
12:11:35.243 DEBUG: 'SFMExport499\Sources\SMEEngLV.pas' indexed with language 'delphi'
12:11:35.243 DEBUG: 'SFMExport499\Sources\SMEEngRZLV.pas' indexed with language 'delphi'
12:11:35.244 DEBUG: 'SFMExport499\Sources\SMEEngSG.pas' indexed with language 'delphi'
12:11:35.244 DEBUG: 'SFMExport499\Sources\SMEEngSM.pas' indexed with language 'delphi'
12:11:35.244 DEBUG: 'SFMExport499\Sources\SMEEngSMI.pas' indexed with language 'delphi'
12:11:35.245 DEBUG: 'SFMExport499\Sources\SMEEngStrings.pas' indexed with language 'delphi'
12:11:35.245 DEBUG: 'SFMExport499\Sources\SMEEngVG.pas' indexed with language 'delphi'
12:11:35.245 DEBUG: 'SFMExport499\Sources\SMEEngWW.pas' indexed with language 'delphi'
12:11:35.245 DEBUG: 'SFMExport499\Sources\SMEMIME.pas' indexed with language 'delphi'
12:11:35.246 DEBUG: 'SFMExport499\Sources\SMEMonitor.dfm' indexed with no language
12:11:35.246 DEBUG: 'SFMExport499\Sources\SMEMonitor.pas' indexed with language 'delphi'
12:11:35.246 DEBUG: 'SFMExport499\Sources\SMEReg.pas' indexed with language 'delphi'
12:11:35.246 DEBUG: 'SFMExport499\Sources\SMESave.dfm' indexed with no language
12:11:35.247 DEBUG: 'SFMExport499\Sources\SMESave.pas' indexed with language 'delphi'
12:11:35.247 DEBUG: 'SFMExport499\Sources\SMESpecs.dfm' indexed with no language
12:11:35.247 DEBUG: 'SFMExport499\Sources\SMESpecs.pas' indexed with language 'delphi'
12:11:35.248 DEBUG: 'SFMExport499\Sources\SMEStat.dfm' indexed with no language
12:11:35.248 DEBUG: 'SFMExport499\Sources\SMEStat.pas' indexed with language 'delphi'
12:11:35.248 DEBUG: 'SFMExport499\Sources\SMEUtils.pas' indexed with language 'delphi'
12:11:35.248 DEBUG: 'SFMExport499\Sources\SMEWiz.dfm' indexed with no language
12:11:35.249 DEBUG: 'SFMExport499\Sources\SMEWiz.pas' indexed with language 'delphi'
12:11:35.249 DEBUG: 'SFMExport499\Sources\smew_cat.res' indexed with no language
12:11:35.249 DEBUG: 'SFMExport499\Sources\SMExportC3.bpk' indexed with no language
12:11:35.249 DEBUG: 'SFMExport499\Sources\SMExportC3.cpp' indexed with no language
12:11:35.250 DEBUG: 'SFMExport499\Sources\SMExportC3.res' indexed with no language
12:11:35.250 DEBUG: 'SFMExport499\Sources\SMExportC4.bpk' indexed with no language
12:11:35.250 DEBUG: 'SFMExport499\Sources\SMExportC4.cpp' indexed with no language
12:11:35.250 DEBUG: 'SFMExport499\Sources\SMExportC4.res' indexed with no language
12:11:35.250 DEBUG: 'SFMExport499\Sources\SMExportC5.bpk' indexed with no language
12:11:35.251 DEBUG: 'SFMExport499\Sources\SMExportC5.cpp' indexed with no language
12:11:35.251 DEBUG: 'SFMExport499\Sources\SMExportC5.res' indexed with no language
12:11:35.251 DEBUG: 'SFMExport499\Sources\SMExportC6.bpk' indexed with no language
12:11:35.251 DEBUG: 'SFMExport499\Sources\SMExportC6.cpp' indexed with no language
12:11:35.252 DEBUG: 'SFMExport499\Sources\SMExportC6.res' indexed with no language
12:11:35.257 DEBUG: 'SFMExport499\Sources\SMExportC6d.bpk' indexed with no language
12:11:35.257 DEBUG: 'SFMExport499\Sources\SMExportC6d.cpp' indexed with no language
12:11:35.258 DEBUG: 'SFMExport499\Sources\SMExportC6d.res' indexed with no language
12:11:35.258 DEBUG: 'SFMExport499\Sources\SMExportC6r.bpk' indexed with no language
12:11:35.258 DEBUG: 'SFMExport499\Sources\SMExportC6r.cpp' indexed with no language
12:11:35.258 DEBUG: 'SFMExport499\Sources\SMExportC6r.res' indexed with no language
12:11:35.259 DEBUG: 'SFMExport499\Sources\SMExportD2005.dof' indexed with no language
12:11:35.259 DEBUG: 'SFMExport499\Sources\SMExportD2005.dpk' indexed with language 'delphi'
12:11:35.259 DEBUG: 'SFMExport499\Sources\SMExportD2005.res' indexed with no language
12:11:35.259 DEBUG: 'SFMExport499\Sources\SMExportD2005d.dpk' indexed with language 'delphi'
12:11:35.260 DEBUG: 'SFMExport499\Sources\SMExportD2005d.res' indexed with no language
12:11:35.260 DEBUG: 'SFMExport499\Sources\SMExportD2005r.dpk' indexed with language 'delphi'
12:11:35.260 DEBUG: 'SFMExport499\Sources\SMExportD2005r.res' indexed with no language
12:11:35.260 DEBUG: 'SFMExport499\Sources\SMExportD2006.dof' indexed with no language
12:11:35.261 DEBUG: 'SFMExport499\Sources\SMExportD2006.dpk' indexed with language 'delphi'
12:11:35.261 DEBUG: 'SFMExport499\Sources\SMExportD2006.res' indexed with no language
12:11:35.261 DEBUG: 'SFMExport499\Sources\SMExportD2006n.dof' indexed with no language
12:11:35.262 DEBUG: 'SFMExport499\Sources\SMExportD2006n.dpk' indexed with language 'delphi'
12:11:35.262 DEBUG: 'SFMExport499\Sources\SMExportD2006n.res' indexed with no language
12:11:35.262 DEBUG: 'SFMExport499\Sources\SMExportD2007.dof' indexed with no language
12:11:35.262 DEBUG: 'SFMExport499\Sources\SMExportD2007.dpk' indexed with language 'delphi'
12:11:35.262 DEBUG: 'SFMExport499\Sources\SMExportD2007.res' indexed with no language
12:11:35.263 DEBUG: 'SFMExport499\Sources\SMExportD2009.dpk' indexed with language 'delphi'
12:11:35.263 DEBUG: 'SFMExport499\Sources\SMExportD2009.res' indexed with no language
12:11:35.263 DEBUG: 'SFMExport499\Sources\SMExportD2010.dpk' indexed with language 'delphi'
12:11:35.263 DEBUG: 'SFMExport499\Sources\SMExportD2010.res' indexed with no language
12:11:35.263 DEBUG: 'SFMExport499\Sources\SMExportD3.dpk' indexed with language 'delphi'
12:11:35.264 DEBUG: 'SFMExport499\Sources\SMExportD3.res' indexed with no language
12:11:35.264 DEBUG: 'SFMExport499\Sources\SMExportD3d.dpk' indexed with language 'delphi'
12:11:35.264 DEBUG: 'SFMExport499\Sources\SMExportD3d.res' indexed with no language
12:11:35.264 DEBUG: 'SFMExport499\Sources\SMExportD3r.dpk' indexed with language 'delphi'
12:11:35.265 DEBUG: 'SFMExport499\Sources\SMExportD3r.res' indexed with no language
12:11:35.265 DEBUG: 'SFMExport499\Sources\SMExportD4.dof' indexed with no language
12:11:35.265 DEBUG: 'SFMExport499\Sources\SMExportD4.dpk' indexed with language 'delphi'
12:11:35.266 DEBUG: 'SFMExport499\Sources\SMExportD4.res' indexed with no language
12:11:35.266 DEBUG: 'SFMExport499\Sources\SMExportd5.dof' indexed with no language
12:11:35.266 DEBUG: 'SFMExport499\Sources\SMExportd5.dpk' indexed with language 'delphi'
12:11:35.266 DEBUG: 'SFMExport499\Sources\SMExportD5.res' indexed with no language
12:11:35.266 DEBUG: 'SFMExport499\Sources\SMExportD6.dof' indexed with no language
12:11:35.267 DEBUG: 'SFMExport499\Sources\SMExportD6.dpk' indexed with language 'delphi'
12:11:35.267 DEBUG: 'SFMExport499\Sources\SMExportD6.res' indexed with no language
12:11:35.267 DEBUG: 'SFMExport499\Sources\SMExportD6d.dpk' indexed with language 'delphi'
12:11:35.267 DEBUG: 'SFMExport499\Sources\SMExportD6d.res' indexed with no language
12:11:35.267 DEBUG: 'SFMExport499\Sources\SMExportD6r.dpk' indexed with language 'delphi'
12:11:35.268 DEBUG: 'SFMExport499\Sources\SMExportD6r.res' indexed with no language
12:11:35.268 DEBUG: 'SFMExport499\Sources\SMExportD7.dof' indexed with no language
12:11:35.274 DEBUG: 'SFMExport499\Sources\SMExportD7.dpk' indexed with language 'delphi'
12:11:35.274 DEBUG: 'SFMExport499\Sources\SMExportD7.res' indexed with no language
12:11:35.274 DEBUG: 'SFMExport499\Sources\SMExportD7d.dpk' indexed with language 'delphi'
12:11:35.275 DEBUG: 'SFMExport499\Sources\SMExportD7d.res' indexed with no language
12:11:35.275 DEBUG: 'SFMExport499\Sources\SMExportD7r.dpk' indexed with language 'delphi'
12:11:35.275 DEBUG: 'SFMExport499\Sources\SMExportD7r.res' indexed with no language
12:11:35.275 DEBUG: 'son.bat' indexed with no language
12:11:35.275 DEBUG: 'TSNameValueList.pas' indexed with language 'delphi'
12:11:35.276 DEBUG: 'UUnitTesting.pas' indexed with language 'delphi'
12:11:35.276 DEBUG: 'UUnitTestingExpressions.pas' indexed with language 'delphi'
12:11:35.276 DEBUG: 'UUnitTestingLocalSessionValue.pas' indexed with language 'delphi'
12:11:35.276 DEBUG: 'UUnitTestingSessionValue.pas' indexed with language 'delphi'
12:11:35.276 DEBUG: 'w.dpr' indexed with language 'delphi'
12:11:35.277 DEBUG: 'w.dproj' indexed with no language
12:11:35.277 DEBUG: 'w.res' indexed with no language
12:11:35.277 DEBUG: 'w2w-pete.dproj' indexed with no language
12:11:35.277 DEBUG: 'w2w-todd-back.dproj' indexed with no language
12:11:35.277 DEBUG: 'w2w.dof' indexed with no language
12:11:35.277 DEBUG: 'w2w.dpr' indexed with language 'delphi'
12:11:35.277 DEBUG: 'w2w.dproj' indexed with no language
12:11:35.277 DEBUG: 'w2w.mes' indexed with no language
12:11:35.277 DEBUG: 'w2w.res' indexed with no language
12:11:35.277 DEBUG: 'w2wBasic.pas' indexed with language 'delphi'
12:11:35.277 DEBUG: 'w2wConfig.pas' indexed with language 'delphi'
12:11:35.278 DEBUG: 'w2wConst.pas' indexed with language 'delphi'
12:11:35.278 DEBUG: 'w2wCore.pas' indexed with language 'delphi'
12:11:35.278 DEBUG: 'w2wCrypt.pas' indexed with language 'delphi'
12:11:35.278 DEBUG: 'w2wDateUtils.pas' indexed with language 'delphi'
12:11:35.278 DEBUG: 'w2wDb.pas' indexed with language 'delphi'
12:11:35.278 DEBUG: 'w2wDbIO.pas' indexed with language 'delphi'
12:11:35.278 DEBUG: 'w2wEmailUtils.pas' indexed with language 'delphi'
12:11:35.278 DEBUG: 'w2wEmpEntry.pas' indexed with language 'delphi'
12:11:35.279 DEBUG: 'w2wExpr.pas' indexed with language 'delphi'
12:11:35.279 DEBUG: 'W2WGroup.groupproj' indexed with no language
12:11:35.279 DEBUG: 'w2wHash.pas' indexed with language 'delphi'
12:11:35.279 DEBUG: 'w2wHTTP.pas' indexed with language 'delphi'
12:11:35.279 DEBUG: 'w2wIAM.pas' indexed with language 'delphi'
12:11:35.280 DEBUG: 'w2wJWK.pas' indexed with language 'delphi'
12:11:35.280 DEBUG: 'w2wJWT.pas' indexed with language 'delphi'
12:11:35.280 DEBUG: 'w2wLog.pas' indexed with language 'delphi'
12:11:35.280 DEBUG: 'w2wMain.dfm' indexed with no language
12:11:35.280 DEBUG: 'w2wMain.pas' indexed with language 'delphi'
12:11:35.281 DEBUG: 'w2wMemLog.pas' indexed with language 'delphi'
12:11:35.281 DEBUG: 'w2wMFA.pas' indexed with language 'delphi'
12:11:35.281 DEBUG: 'w2wProgress.pas' indexed with language 'delphi'
12:11:35.281 DEBUG: 'w2wSessions.pas' indexed with language 'delphi'
12:11:35.281 DEBUG: 'w2wSqlLog.pas' indexed with language 'delphi'
12:11:35.282 DEBUG: 'w2wSSO.pas' indexed with language 'delphi'
12:11:35.282 DEBUG: 'w2wSSOFactory.pas' indexed with language 'delphi'
12:11:35.282 DEBUG: 'w2wSSO_Azure.pas' indexed with language 'delphi'
12:11:35.282 DEBUG: 'w2wSSO_Facebook.pas' indexed with language 'delphi'
12:11:35.282 DEBUG: 'w2wSSO_Google.pas' indexed with language 'delphi'
12:11:35.282 DEBUG: 'w2wSSO_Microsoft.pas' indexed with language 'delphi'
12:11:35.282 DEBUG: 'w2wSSO_Okta.pas' indexed with language 'delphi'
12:11:35.283 DEBUG: 'w2wSys.pas' indexed with language 'delphi'
12:11:35.283 DEBUG: 'w2wTEST_Icon.ico' indexed with no language
12:11:35.283 DEBUG: 'w2wTypes.pas' indexed with language 'delphi'
12:11:35.283 DEBUG: 'w2wVars.pas' indexed with language 'delphi'
12:11:35.284 DEBUG: 'xMadBasicSources/mad-ORIGINAL.inc' generated metadata with charset 'windows-1252'
12:11:35.290 DEBUG: 'xMadBasicSources\mad-ORIGINAL.inc' indexed with language 'php'
12:11:35.291 DEBUG: 'xMadBasicSources/mad.inc' generated metadata with charset 'windows-1252'
12:11:35.291 DEBUG: 'xMadBasicSources\mad.inc' indexed with language 'php'
12:11:35.291 DEBUG: 'xMadBasicSources\madBasic.pas' indexed with language 'delphi'
12:11:35.291 DEBUG: 'xMadBasicSources\madBasic_.dpk' indexed with language 'delphi'
12:11:35.292 DEBUG: 'xMadBasicSources\madCrypt.pas' indexed with language 'delphi'
12:11:35.292 DEBUG: 'xMadBasicSources\madGraphics.pas' indexed with language 'delphi'
12:11:35.292 DEBUG: 'xMadBasicSources\madHelp.pas' indexed with language 'delphi'
12:11:35.292 DEBUG: 'xMadBasicSources\madHelp_.dpk' indexed with language 'delphi'
12:11:35.292 DEBUG: 'xMadBasicSources\madLists.pas' indexed with language 'delphi'
12:11:35.293 DEBUG: 'xMadBasicSources\madRes.pas' indexed with language 'delphi'
12:11:35.293 DEBUG: 'xMadBasicSources\madStrings - MODIFIED BY PLR.pas' indexed with language 'delphi'
12:11:35.293 DEBUG: 'xMadBasicSources\madStrings-ORIGINAL.pas' indexed with language 'delphi'
12:11:35.293 DEBUG: 'xMadBasicSources\madStrings.pas' indexed with language 'delphi'
12:11:35.294 DEBUG: 'xMadBasicSources\madTools.pas' indexed with language 'delphi'
12:11:35.294 DEBUG: 'xMadBasicSources\madTypes.pas' indexed with language 'delphi'
12:11:35.294 DEBUG: 'xMadBasicSources\madZip.pas' indexed with language 'delphi'
12:11:35.295 INFO: 360 files indexed
12:11:35.296 INFO: Quality profile for delphi: Sonar way
12:11:35.296 INFO: Quality profile for php: Sonar way
12:11:35.296 INFO: ------------- Run sensors on module Jantest
12:11:35.335 INFO: Load metrics repository
12:11:35.347 DEBUG: GET 200 http://localhost:9000/api/metrics/search?ps=500&p=1 | time=9ms
12:11:35.356 INFO: Load metrics repository (done) | time=21ms
12:11:36.104 DEBUG: 'Import external issues report' skipped because one of the required properties is missing
12:11:36.106 DEBUG: 'Python Sensor' skipped because there is no related file in current project
12:11:36.106 DEBUG: 'Cobertura Sensor for Python coverage' skipped because there is no related file in current project
12:11:36.107 DEBUG: 'PythonXUnitSensor' skipped because there is no related file in current project
12:11:36.107 DEBUG: 'Import of Pylint issues' skipped because there is no related file in current project
12:11:36.107 DEBUG: 'Import of Bandit issues' skipped because there is no related file in current project
12:11:36.108 DEBUG: 'Import of Flake8 issues' skipped because there is no related file in current project
12:11:36.108 DEBUG: 'Import of Mypy issues' skipped because there is no related file in current project
12:11:36.108 DEBUG: 'Import of Ruff issues' skipped because there is no related file in current project
12:11:36.108 DEBUG: 'XML Sensor' skipped because there is no related file in current project
12:11:36.109 DEBUG: 'Import of PHPStan issues' skipped because one of the required properties is missing
12:11:36.110 DEBUG: 'Import of Psalm issues' skipped because one of the required properties is missing
12:11:36.110 DEBUG: 'IaC Terraform Sensor' skipped because there is no related file in current project
12:11:36.110 DEBUG: 'IaC CloudFormation Sensor' skipped because there is no related file in current project
12:11:36.111 DEBUG: 'IaC Kubernetes Sensor' skipped because there is no related file in current project
12:11:36.111 DEBUG: 'IaC AzureResourceManager Sensor' skipped because there is no related file in current project
12:11:36.112 DEBUG: 'Generic Test Executions Report' skipped because one of the required properties is missing
12:11:36.113 DEBUG: Sensors : HTML -> DelphiSensor -> DelphiCoverageSensor -> Delphi NUnit Sensor -> JaCoCo XML Report Importer -> PHP sensor -> Analyzer for "php.ini" files -> PHPUnit report sensor -> IaC Docker Sensor -> TextAndSecretsSensor
12:11:36.113 INFO: Sensor HTML [web]
12:11:36.135 INFO: Sensor HTML [web] (done) | time=22ms
12:11:36.135 INFO: Sensor DelphiSensor [communitydelphi]
12:11:36.137 INFO: Platform: WINDOWS
12:11:36.137 INFO: Architecture: X86
12:11:36.138 INFO: Compiler version: 35.0
12:11:36.139 INFO: Indexing 6 dproj file(s) and 1 groupproj file(s)...
12:11:36.310 WARN: Invalid DCC_UnitSearchPath directory: c:\dev\w2w-main-dll\madBasicSources
12:11:36.310 WARN: Invalid DCC_UnitSearchPath directory: c:\dev\madCollection\madExcept\Sources
12:11:36.312 WARN: Invalid DCC_UnitSearchPath directory: c:\dev\madCollection\madDisAsm\Sources
12:11:36.319 INFO: ------------------------------------------------------------------------
12:11:36.319 INFO: EXECUTION FAILURE
12:11:36.319 INFO: ------------------------------------------------------------------------
12:11:36.320 INFO: Total time: 6.383s
12:11:36.345 INFO: Final Memory: 15M/80M
12:11:36.345 INFO: ------------------------------------------------------------------------
12:11:36.347 ERROR: Error during SonarScanner execution
java.nio.file.InvalidPathException: Illegal char <:> at index 2: xc:/dev/w2w-main-dll/madBasicSources
        at java.base/sun.nio.fs.WindowsPathParser.normalize(Unknown Source)
        at java.base/sun.nio.fs.WindowsPathParser.parse(Unknown Source)
        at java.base/sun.nio.fs.WindowsPathParser.parse(Unknown Source)
        at java.base/sun.nio.fs.WindowsPath.parse(Unknown Source)
        at java.base/sun.nio.fs.WindowsFileSystem.getPath(Unknown Source)
        at java.base/java.nio.file.Path.of(Unknown Source)
        at au.com.integradev.delphi.msbuild.DelphiProjectParser.resolvePath(DelphiProjectParser.java:135)
        at java.base/java.util.stream.ReferencePipeline$3$1.accept(Unknown Source)
        at java.base/java.util.stream.ReferencePipeline$3$1.accept(Unknown Source)
        at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(Unknown Source)
        at java.base/java.util.stream.AbstractPipeline.copyInto(Unknown Source)
        at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source)
        at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(Unknown Source)
        at java.base/java.util.stream.AbstractPipeline.evaluate(Unknown Source)
        at java.base/java.util.stream.ReferencePipeline.collect(Unknown Source)
        at au.com.integradev.delphi.msbuild.DelphiProjectParser.createPathList(DelphiProjectParser.java:107)
        at au.com.integradev.delphi.msbuild.DelphiProjectParser.createSearchDirectories(DelphiProjectParser.java:83)
        at au.com.integradev.delphi.msbuild.DelphiProjectParser.parse(DelphiProjectParser.java:66)
        at au.com.integradev.delphi.msbuild.DelphiProjectHelper.indexProject(DelphiProjectHelper.java:242)
        at java.base/java.util.ArrayList.forEach(Unknown Source)
        at au.com.integradev.delphi.msbuild.DelphiProjectHelper.indexProjects(DelphiProjectHelper.java:198)
        at au.com.integradev.delphi.msbuild.DelphiProjectHelper.getConditionalDefines(DelphiProjectHelper.java:326)
        at au.com.integradev.delphi.DelphiSensor.executeOnFiles(DelphiSensor.java:95)
        at au.com.integradev.delphi.DelphiSensor.execute(DelphiSensor.java:83)
        at org.sonar.scanner.sensor.AbstractSensorWrapper.analyse(AbstractSensorWrapper.java:64)
        at org.sonar.scanner.sensor.ModuleSensorsExecutor.execute(ModuleSensorsExecutor.java:88)
        at org.sonar.scanner.sensor.ModuleSensorsExecutor.lambda$execute$1(ModuleSensorsExecutor.java:61)
        at org.sonar.scanner.sensor.ModuleSensorsExecutor.withModuleStrategy(ModuleSensorsExecutor.java:79)
        at org.sonar.scanner.sensor.ModuleSensorsExecutor.execute(ModuleSensorsExecutor.java:61)
        at org.sonar.scanner.scan.SpringModuleScanContainer.doAfterStart(SpringModuleScanContainer.java:82)
        at org.sonar.core.platform.SpringComponentContainer.startComponents(SpringComponentContainer.java:226)
        at org.sonar.core.platform.SpringComponentContainer.execute(SpringComponentContainer.java:205)
        at org.sonar.scanner.scan.SpringProjectScanContainer.scan(SpringProjectScanContainer.java:204)
        at org.sonar.scanner.scan.SpringProjectScanContainer.scanRecursively(SpringProjectScanContainer.java:200)
        at org.sonar.scanner.scan.SpringProjectScanContainer.doAfterStart(SpringProjectScanContainer.java:173)
        at org.sonar.core.platform.SpringComponentContainer.startComponents(SpringComponentContainer.java:226)
        at org.sonar.core.platform.SpringComponentContainer.execute(SpringComponentContainer.java:205)                                         at org.sonar.scanner.bootstrap.SpringScannerContainer.doAfterStart(SpringScannerContainer.java:351)                    12:12 PM        at org.sonar.core.platform.SpringComponentContainer.startComponents(SpringComponentContainer.java:226)                                 at org.sonar.core.platform.SpringComponentContainer.execute(SpringComponentContainer.java:205)                                         at org.sonar.scanner.bootstrap.SpringGlobalContainer.doAfterStart(SpringGlobalContainer.java:138)                                      at org.sonar.core.platform.SpringComponentContainer.startComponents(SpringComponentContainer.java:226)                                 at org.sonar.core.platform.SpringComponentContainer.execute(SpringComponentContainer.java:205)                                         at org.sonar.batch.bootstrapper.Batch.doExecute(Batch.java:71)                                                                         at org.sonar.batch.bootstrapper.Batch.execute(Batch.java:65)                                                                           at org.sonarsource.scanner.api.internal.batch.BatchIsolatedLauncher.execute(BatchIsolatedLauncher.java:46)                             at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)                                                      at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)                                                      at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)                                                  at java.base/java.lang.reflect.Method.invoke(Unknown Source)                                                                           at org.sonarsource.scanner.api.internal.IsolatedLauncherProxy.invoke(IsolatedLauncherProxy.java:60)                                    at jdk.proxy1/jdk.proxy1.$Proxy0.execute(Unknown Source)                                                                               at org.sonarsource.scanner.api.EmbeddedScanner.doExecute(EmbeddedScanner.java:189)                                                     at org.sonarsource.scanner.api.EmbeddedScanner.execute(EmbeddedScanner.java:138)                                                       at org.sonarsource.scanner.cli.Main.execute(Main.java:126)                                                                             at org.sonarsource.scanner.cli.Main.execute(Main.java:81)                                                                              at org.sonarsource.scanner.cli.Main.main(Main.java:62)                                                                         Exception in thread "Thread-0" java.lang.NoClassDefFoundError: ch/qos/logback/classic/spi/ThrowableProxy                                       at ch.qos.logback.classic.spi.LoggingEvent.<init>(LoggingEvent.java:145)                                                               at ch.qos.logback.classic.Logger.buildLoggingEventAndAppend(Logger.java:424)                                                           at ch.qos.logback.classic.Logger.filterAndLog_0_Or3Plus(Logger.java:386)                                                               at ch.qos.logback.classic.Logger.error(Logger.java:543)                                                                                at org.eclipse.jgit.internal.util.ShutdownHook.cleanup(ShutdownHook.java:87)                                                           at java.base/java.lang.Thread.run(Unknown Source)                                                                              Caused by: java.lang.ClassNotFoundException: ch.qos.logback.classic.spi.ThrowableProxy                                                         at java.base/java.net.URLClassLoader.findClass(Unknown Source)                                                                         at org.sonarsource.scanner.api.internal.IsolatedClassloader.loadClass(IsolatedClassloader.java:82)                                     at java.base/java.lang.ClassLoader.loadClass(Unknown Source)                                                                           ... 6 more 

### Steps to reproduce

Executing the following

sonar-scanner.bat -X -D"sonar.projectKey=Jantest" -D"sonar.sources=." -D"sonar.host.url=http://localhost:9000" -D"sonar.token=sqp_8607858a1d68745fca4e8c2ec1cad1c4d62f6fc9" 

Please let me know what you need and I will provide it to you 


### Minimal Delphi code exhibiting the issue

_No response_

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -20,6 +20,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - File pointer errors when an issue is raised on the first line of a file with a byte order mark.
 - Symbol table construction errors for program files omitting the `program` header.
 - Parsing errors on parenthesised anonymous methods.
+- Scan failures on invalid paths in dproj files.
 - Incorrect ordering of edits in the "Separate grouped parameters" quick fix for `GroupedParameterDeclaration`.
 - SonarLint registration of rules that don't support execution in a SonarLint context.
 

--- a/delphi-frontend/src/main/java/au/com/integradev/delphi/msbuild/DelphiProjectParser.java
+++ b/delphi-frontend/src/main/java/au/com/integradev/delphi/msbuild/DelphiProjectParser.java
@@ -28,6 +28,7 @@ import com.google.common.base.Splitter;
 import com.google.common.collect.ImmutableSortedMap;
 import com.google.common.collect.ImmutableSortedSet;
 import java.nio.file.Files;
+import java.nio.file.InvalidPathException;
 import java.nio.file.Path;
 import java.util.ArrayList;
 import java.util.Collections;
@@ -35,7 +36,7 @@ import java.util.List;
 import java.util.Map;
 import java.util.Set;
 import java.util.TreeMap;
-import java.util.stream.Collectors;
+import javax.annotation.Nullable;
 import org.apache.commons.lang3.StringUtils;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
@@ -93,18 +94,32 @@ final class DelphiProjectParser {
   }
 
   private List<Path> createPathList(ProjectProperties properties, String propertyName) {
-    return propertyList(properties.get(propertyName)).stream()
-        .map(DelphiUtils::normalizeFileName)
-        .map(this::resolvePath)
-        .filter(
-            directory -> {
-              if (!Files.exists(directory) || !Files.isDirectory(directory)) {
-                LOG.warn("Invalid {} directory: {}", propertyName, directory);
-                return false;
+    List<Path> result = new ArrayList<>();
+    propertyList(properties.get(propertyName))
+        .forEach(
+            pathString -> {
+              Path path = resolveDirectory(pathString);
+              if (path == null) {
+                LOG.warn("Invalid {} directory: {}", propertyName, pathString);
+                return;
               }
-              return true;
-            })
-        .collect(Collectors.toUnmodifiableList());
+              result.add(path);
+            });
+    return Collections.unmodifiableList(result);
+  }
+
+  @Nullable
+  private Path resolveDirectory(String pathString) {
+    try {
+      pathString = DelphiUtils.normalizeFileName(pathString);
+      Path path = DelphiUtils.resolvePathFromBaseDir(evaluationDirectory(), Path.of(pathString));
+      if (Files.isDirectory(path)) {
+        return path;
+      }
+    } catch (InvalidPathException e) {
+      LOG.debug("Invalid path string", e);
+    }
+    return null;
   }
 
   private static Map<String, String> createUnitAliases(ProjectProperties properties) {
@@ -129,10 +144,6 @@ final class DelphiProjectParser {
       return Collections.emptyList();
     }
     return Splitter.on(';').omitEmptyStrings().splitToList(value);
-  }
-
-  private Path resolvePath(String pathString) {
-    return DelphiUtils.resolvePathFromBaseDir(evaluationDirectory(), Path.of(pathString));
   }
 
   private Path evaluationDirectory() {

